cypress/integration/team/teams_spec.js

```diff
@@ -112,7 +112,7 @@ describe('Teams Suite', () => {
 
     it('TS14633 Leave all teams', () => {
         // 1. Login and go to /
-        cy.apiLogin('user-2');
+        cy.apiLogin('user-3');
```

Comment:
Not related to the issue but changed anyway.  Since the original user-2 is leaving teams, other tests that's using such user will fail (in case this test runs before that).

Long term might be to have this test rejoin the teams on after hook.  Will address that separately.